### ISSUE TITLE
DocLinkChecker: Support for `%20` in links

### ISSUE THREAD

**Issue by Fabian-Schmidt** [Ellerbach/docfx-companion-tools#60]
DocLinkChecker will report an error if a Markdown contains a link to a file which contains an escaped space `%20`. It can validate if the corresponding file exists after URL decoding the link.

Example:

```markdown
[okay](okay.txt)
[not okay](not%20okay.txt)
```

```txt
$ ls
index.md
okay.txt
not okay.txt
```

DocLinkChecker will report `***ERROR Not found: not%20okay.txt`. But the file does exist and the link is fine.

**Comment by Ellerbach** [comment]
You are correct. I just checked and it's authorized to have encoded URL in Markdown link. Adding in the bug label.
If you want as well to contribute, we ❤️ PR 😊
